### ISSUE TITLE
External links accessibility issue "Links are not clearly identifiable"

### ISSUE THREAD

**Issue by josenino95** [carpentries/varnish#191]
As #190, I'm also raising this issue after an accessibility review done with the SiteImprove tool.

The tool is flagging all hyperlinks with `class="external-link`" with issues. Here is SiteImprove's description of the issue

> ### Links are not clearly identifiable
> Color alone is not enough to distinguish a link from the surrounding text.
>

> ### Why is this an issue?
> People who are colorblind may be unable to find a link if color is its only distinguishing feature.
> 
> Providing another visual cue —such as underlining or bolding the link text— will also make it easier for non-colorblind people (especially people with low vision) to scan the page.
> 
> WCAG 2.2: [Understanding Success Criterion 1.4.1: Use of Color](https://www.w3.org/WAI/WCAG22/Understanding/use-of-color)
> Siteimprove Alfa: [Technical documentation for rule SIA-R62](https://alfa.siteimprove.com/rules/sia-r62)

After a Google search and some reading on my side, I see that, [for WCAG, only a 3:1 contrast](https://www.w3.org/WAI/WCAG22/Techniques/general/G183) between link and normal text color is enough for compliance. However, my quick review of the Workbench template tells me this is not accomplished:
-  In Light mode, the contrast is only [1.56:1](https://webaim.org/resources/contrastchecker/?fcolor=0044D7&bcolor=383838) between #0044D7 (blue/link) and #383838 (black/normal text).
- In Dark mode, the contrast is [1.54:1](https://webaim.org/resources/contrastchecker/?fcolor=8BB9FE&bcolor=DEE2E6) between #8BB9FE (blue/link) and #DEE2E6 (black/normal text)

[SiteImprove's recommendation](https://webaim.org/resources/contrastchecker/?fcolor=0044D7&bcolor=383838) is stronger, and it advises to have links differentiable not only by color.
